include/CMakeLists.txt

```diff
@@ -95,6 +95,7 @@ if(UNIX)
     set(link_platform_HEADERS
       ${link_platform_HEADERS}
       ${link_platform_DIR}/linux/Linux.hpp
+      ${link_platform_DIR}/linux/Clock.hpp
```

Comment:
@abique: please insert this new without breaking the alphabetical sorting, thanks!